worker/workercmd/containerd.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/tedsuo/ifrit/grouper"
 )
 
-const containerdNamespace = "concourse"
+var containerdNamespace = "concourse" + os.Getenv("CONCOURSE_CONTAINERD_NAMESPACE_SUFFIX")
```

Comment:
Thanks for the PR!

This implementation makes it hard for anyone to discover that this flag even exists. Also what if someone likes setting all their stuff with CLI flags instead of env vars? Overall, we should probably add this alongside the rest of the configuration fields that Concourse provides.

Could you add this as a new field on this struct:
https://github.com/concourse/concourse/blob/da6b3b68bc56884498cce9be31f6231b1d384ef2/worker/workercmd/worker_linux.go#L40

Then instead of directly calling `os.GetEnv` you can pull the suffix from the `ContainerdRuntime` struct. Probably around here, where the namespace is passed along:
https://github.com/concourse/concourse/blob/da6b3b68bc56884498cce9be31f6231b1d384ef2/worker/workercmd/containerd.go#L74-L77

Reply:
@analytically Do you need any help with making the changes?